src/pages/settings/schedules/common/components/EmailStatement.tsx

```diff
@@ -48,11 +48,20 @@ export function EmailStatement(props: Props) {
 
   const [selectedClients, setSelectedClients] = useState<Client[]>([]);
 
+  const handleChangeParameters = (clients: Client[]) => {
```

Comment:
@beganovich After David reported bugs related to Schedules during testing, I found an additional one. So, this is just a quick fix for that one.